### ISSUE TITLE
Typos in our release notes

### ISSUE THREAD

**Issue by philderbeast** [haskell/cabal#10604]
A follow on from #10601. Running the typos spell checker on `*.md` files shows that we have typos in our release notes. Can these be fixed for past releases?

```
$ typos release-notes/*.md
error: `informations` should be `information`
  --> release-notes/Cabal-3.10.1.0.md:49:65
   |
49 |   - Add `PackageInfo_` module to embed portable package-related informations (issue #3909)
   |                                                                 ^^^^^^^^^^^^
   |
error: `dependenices` should be `dependencies`
  --> release-notes/Cabal-3.12.0.0.md:39:48
   |
39 |     to load into the repl without building any dependenices which we will load
   |                                                ^^^^^^^^^^^^
   |
error: `satisifed` should be `satisfied`
  --> release-notes/Cabal-3.12.0.0.md:40:49
   |
40 |     in the same session, because the promise is satisifed due to loading the
   |                                                 ^^^^^^^^^
   |
error: `mimick` should be `mimic`
  --> release-notes/Cabal-3.12.0.0.md:70:6
   |
70 |   To mimick `initialBuildSteps` behaviour when there is no custom Setup, you
   |      ^^^^^^
   |
error: `parrallel` should be `parallel`
  --> release-notes/Cabal-3.12.0.0.md:147:27
    |
147 |        when GHC is run in parrallel make mode ('--ghc-options -j').
    |                           ^^^^^^^^^
    |
error: `directiory` should be `directory`
  --> release-notes/Cabal-3.12.0.0.md:169:16
    |
169 |     flags to a directiory. The option is called `--repl-multi-file <DIR>`.
    |                ^^^^^^^^^^
    |
error: `insonsistency` should be `inconsistency`
  --> release-notes/Cabal-3.12.0.0.md:267:10
    |
267 |     Such insonsistency seems to be always a mistake, and it's easy to fix once
    |          ^^^^^^^^^^^^^
    |
error: `unparseable` should be `unparsable`
  --> release-notes/cabal-install-3.10.3.0.md:36:16
   |
36 |   ignored, and unparseable package names are considered nonexistent.
   |                ^^^^^^^^^^^
   |
error: `paralellism` should be `parallelism`
  --> release-notes/cabal-install-3.12.1.0.md:143:22
    |
143 |     control how much paralellism it uses, coordinating with other simultaneously running
    |                      ^^^^^^^^^^^
    |
```

**Comment by philderbeast** [comment]
@geekosaur and @ulysses4ever, from https://github.com/haskell/cabal/pull/10603#issuecomment-2506797669, can we fix the typos now in `release-notes/*.md`?

We should check files in `changelog.d` on each pull request but could neglect `release-notes/*.md` files, once they're fixed for spelling.

**Comment by geekosaur** [comment]
I would say fix them now, unless there's a way to tell it to leave alone anything from before 3.14 on the grounds that it's really too late to do anything about them.

**Comment by ulysses4ever** [comment]
> @geekosaur and @ulysses4ever, from https://github.com/haskell/cabal/pull/10603#issuecomment-2506797669, can we fix the typos now in release-notes/*.md?

we can and should.

Note that when I'm talking about adding MD in the CI check, I mean all MD, and release notes are not the only MD files we have.

**Comment by philderbeast** [comment]
Fixed by #10603.
